src/pages/BorrowerProfile/FundedBorrowerProfile.vue

```diff
@@ -196,13 +376,106 @@ export default {
 				});
 			}
 		},
+		fetchLoanData() {
```

Comment:
This function should be called again when it finishes if the preBottom ref is still in the viewport

Reply:
Good catch! Thank you